bool STOP_OPERATIONAL 
---
bool success
